frontend/test/cypress/integration/product/add_to_cart.cy.ts

```diff
@@ -1,138 +1,136 @@
 describe('product page add to cart', () => {
-   const user = cy;
```

Comment:
Should we do this for the rest of the cypress tests too?

Reply:
yea probably. I only did this one since the others didn't seem flaky at the moment